spec/services/scrapers/district_attorney_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+require 'byebug'
+
+RSpec.describe Scrapers::DistrictAttorney do
+  describe '#perform' do
+    fixture_path = 'spec/fixtures/district_attorney.html'
+    html = File.read(fixture_path)
+
+    it 'saves  district attorney and hooks up counties' do
+      # Beaver, Cimmarron, Harper, Texas
+      # Beckham, Custer, Ellis, Roger Mills, Washita
+      # Greer, Harmon, Jackson, Kiowa, Tillman,
+
+      create(:county, name: 'Beaver')
+      create(:county, name: 'Cimarron')
+      create(:county, name: 'Harper')
+      create(:county, name: 'Texas')
+
+      create(:county, name: 'Beckham')
+      create(:county, name: 'Custer')
+      create(:county, name: 'Ellis')
+      create(:county, name: 'Roger Mills')
+      create(:county, name: 'Washita')
+
+      create(:county, name: 'Greer')
+      create(:county, name: 'Harmon')
+      create(:county, name: 'Jackson')
+      create(:county, name: 'Kiowa')
+      create(:county, name: 'Tillman')
+
+      parser = OscnScraper::Parsers::DistrictAttorney.new(html)
+      parser.perform
+
+      data = parser.district_attorneys
+      test = described_class.new(data)
+      test.perform
+
+      expect(County.all.first.district_attorney_id).to eq DistrictAttorney.all.first.id
+      expect(DistrictAttorney.all.size).to eq data.size
+      expect(DistrictAttorney.all.first.name).to eq data.first[:district_attorney]
```

Comment:
Similar to the first expect comment. I think that using `.first` in a database call could make this test flaky in the future. My preference would be explicitly defining the district attorney that we are looking for